### DISCUSSION TITLE
For every conversion type, remember the last units used

### DISCUSSION THREAD

**Discussion by zszabo2000** [Yet-Zio/yetCalc#50] [Ideas]
Details: When the app is run and a particular unit of measure is selected in the "To" or "From" field under any of the measurements tabs, then that unit is automatically stored as the new default for that field/measurement type.

Optional: Create a check box under each measurement type which, when checked, stores the To and From units as the defaults for that measurement type. These defaults will be automatically loaded when the app is started.

Reason:
1. It is likely that a user who runs the app for its conversion function will want to do the same kind of conversion across application sessions. The simplest way requiring the fewest changes would be to automatically store the unit of measure for any field whenever it's changed and automatically load these settings during startup.
2. Optionally, a check box could be added that stores the unit settings for each measurement type as the new defaults only if the box is checked. For clarity, the description underneath the box should read something like "store the units in every category as the new defaults".

**Comment by Yet-Zio** [comment]
I tried to add that while working on #14 as mentioned in that issue. But I ran into some trouble(spinner/selection box initialization problems) and could only provide a setting for remembering the last used section/tab.

I do want to add this as some units have a large list and its hard to keep remembering what you used last. Your second idea about a checkbox option might be a way of providing this feature or I might try the old way, this time working on what was the issue between the fragment and the spinner.

Anyway, @zszabo2000 thanks for your idea. I will probably work on it in the near future.

**Comment by Yet-Zio** [comment]
Done as of #147
